.env.example

```diff
@@ -6,6 +6,7 @@ MAILGUN_API_KEY=""
 MAILGUN_DOMAIN=""
 SECRET_KEY_BASE=""
 SECRET_KEY_JWT=""
+SITE_NAME=SweepAround.Us
```

Comment:
SITE_NAME="We The Sweeple"

And for the logo, I think it'd make sense to jam the words up against each other like you had for the SweepAround.Us logo.

Reply:
Agreed!

Reply:
Done! Just pushed a commit.